src/lib/store.js

```diff
@@ -27,11 +27,13 @@ export default class Store {
 		tcModel.cmpVersion = cmpVersion;
 		tcModel.isServiceSpecific = true;
 		tcModel.supportOOB = false;
+		tcModel.publisherCountryCode = config.publisherCountryCode;
 
 		// decoding to check if string is compatible
 		const decodedConsentString = decodeConsentData(consentString);
-		this.persistedConsentString = decodedConsentString ? consentString : '';
-		this.persistedConsentData = decodedConsentString || {};
+		const isTCFv2Compatible = decodedConsentString && decodedConsentString.version > 1;
```

Comment:
This is needed after libs update, caused by:
https://github.com/InteractiveAdvertisingBureau/iabtcf-es/pull/157/files